5/run-client-docker.sh

```diff
@@ -3,7 +3,7 @@
 # 2020-12-07 PJ:
 # Code below gets local IP for dockerized applications to function and communicate correctly.
 ADVERTISEDIP=`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'`
-GATEWAYPORT=3000
+GATEWAYPORT=3001
 
-docker build -t client -f ./ops/Client/Dockerfile ./ &&
-  docker run -it -e ADVERTISEDIP=$ADVERTISEDIP -e GATEWAYPORT=$GATEWAYPORT --rm client
+docker build -t client-cluster -f ./ops/Client/Dockerfile ./ &&
+  docker run -d -e ADVERTISEDIP=$ADVERTISEDIP -e GATEWAYPORT=$GATEWAYPORT --rm client-cluster
```

Comment:
I think that now we can revisit all docker convenience scripts (silos and clients, all examples) and make sure the containers run with the `-it` flag. Running client in the background does not necessarily make most sense in all scenarios. Will you even see the console output of the client if you run it like this?

Reply:
Clustered scenario will break

    the input device is not a TTY
    the input device is not a TTY

Reply:
Hm. How do you run this? This works: https://github.com/PiotrJustyna/road-to-orleans/blob/main/3/run-client-docker.sh so are you talking about running this from `run-demo-clients.sh`? I would like us to be consistent, so this script should work just as the one in solution 3 (so that it can be executed in isolation, in foreground with visible console output).

I missed it, but solution 4 suffers from the same problem: https://github.com/PiotrJustyna/road-to-orleans/blob/main/4/run-client-docker.sh

Reply:
oh yep, you're right... Just left over. Will correct now